build.gradle

```diff
@@ -46,15 +46,7 @@ android {
         testFunctionalTest true
     }
 
-    // Without this parameter, all builds fail with error:
-    //     Execution failed for task ':mergeArmeabiDebugResources'.
-    //     > Some file crunching failed, see logs for details
-    // https://code.google.com/p/android/issues/detail?id=163378
-    //
-    // NOTE: Setting this causes a crash when opening the DeviceFragment:
-    //       Resources$NotFoundException: File res/drawable/ic_tag_text_outline_black_24dp.xml
-    //       This means we can't use build tools 2.0 until this is fixed.
-    //aaptOptions.setProperty("cruncherEnabled", false)
+    aaptOptions.setProperty("cruncherEnabled", false)
```

Comment:
Did you check if this is still needed? If yes, you should leave that part of the comment.